test/functional/tm/sys/test_sys_application.py

```diff
@@ -53,19 +56,22 @@ def setup_template_test(request, bigip, name, partition):
 
     def teardown():
         delete_resource(test_template)
+
     request.addfinalizer(teardown)
     test_template = template_s.template.create(
         name=name,
         partition=partition,
         actions=definition
     )
+    assert isinstance(test_template, Template)
```

Comment:
I'd like to see something like:

`assert id(test_template) != id(template_s.template)`